CONTRIBUTING.md

```diff
@@ -33,6 +33,59 @@ For example, see [mongo-go-driver tests configuration](https://github.com/Ferret
 `TEST` environment variable should have the value `mongo-go-driver`, or be empty.
 It defines what test configuration to run; empty value runs all configurations.
 
+### Running custom tests
+
+In order to test your application with FerretDB you must use the `command` runner and add your repository as a submodule.
+The `command` runner will invoke any command and CLI arguments.
+
+For example if you want to test your Node.js application with FerretDB, you would do the following:
+
+1. Add the submodule to dance `git submodule add https://github.com/my-app`
+2. Create a shell script in the `tests` directory called `my-app-runtime.sh` with the required logic needed for your test
+3. Create a YAML file called `my-app.yml` in the `tests` directory and provide the `args` field with the shell script so that the runner can invoke it
+4. Start the environment `bin/task env-up DB=ferretdb`
+5. Run the test `bin/task dance DB=ferretdb TEST=my-app`
+
+#### Shell script
```

Comment:
I think we should point to existing example instead of having another copy of it there

Reply:
And the YAML too?

Reply:
Why not? It also exist and we also don't want to make another copy of it

Reply:
Done, I used the java-example as a reference

Reply:
I'm unsure what is "done", but the example is still in the text. We point to existing examples in addition to providing one there, not _instead_. And we should not have them in two places.

Reply:
I see, so just link to examples and not remove all the text related to examples?

Reply:
@AlekSi PTAL.